src/email/index.js

```diff
@@ -20,8 +20,9 @@ const transporter = nodemailer.createTransport({
  */
 const sendMail = async (to, subject, html, text) => {
   try {
+    const fromAddress = 'Molly.Martin@baltimorecity.gov'
     await transporter.sendMail({
-      from: `"Healthcare Roll Call" <${process.env.SMTP_USER}>`, // sender address
+      from: `"Healthcare Roll Call" <${fromAddress}>`, // sender address
```

Comment:
Why was this nessecary?  Seems like you'd want to have the address externalized.  Just seeking to understand.